src/ui/core/StatusBarManager.ts

```
@@ -0,0 +1,482 @@
+        }
+        break;
+    }
+  }
+
+  /**
+   * Handle proxy status change notifications
+   */
+  private handleProxyStatusNotifications(proxy: ProxyTreeItem): void {
+    switch (proxy.status) {
+      case 'error':
+        const assignedKeyCount = proxy.assignedKeys?.length || 0;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)